front-end/src/visitor/VisitorAccessor.cpp

```diff
@@ -78,6 +79,21 @@ antlrcpp::Any StartVisitor::visitAccessorFunction(MapleGrammarParser::AccessorFu
     if (auto symbol = currentSymbolTable->lookup(name)) {
         if (auto symbolFun = dynamic_cast<FunctionDefinition *>(symbol->getDeclaration())) {
 
+            vector<FunctionParam*> argument = visit(ctx->argumentList());
+            if (argument.size() != symbolFun->getParams().size()) {
+                cerr << "Function declaration has not the same number of arguments as Function definition" << endl;
```

Comment:
Can we have a better message with "Too much arguments" or "Not enough arguments"